modules/apps/27-interchain-accounts/host/keeper/relay_test.go

```diff
@@ -49,6 +50,9 @@ func (suite *KeeperTestSuite) TestOnRecvPacket() {
 				}
 
 				packetData = icaPacketData.GetBytes()
+
+				expParams := types.NewParams(true, []string{sdk.MsgTypeURL(msg)})
+				suite.chainB.GetSimApp().ICAHostKeeper.SetParams(suite.chainB.GetContext(), expParams)
```

Comment:
ditto on renaming `expParams` to `params`?

Reply:
Resolved all of these, copy pasta from another file :))